pwiz_tools/Skyline/TestUtil/AssertEx.cs

```diff
@@ -236,6 +236,28 @@ public static void Contains(string value, params string[] parts)
             }
         }
 
+        // Checks that string value could have been partially created with given format
+        // e.g. value "Oh no! My dog has fleas! Now what?" agrees with format "My {0} has {1}!" because it contains
+        // "My ", " has ", and "!" in that order.
+        public static void ContainsSimilar(string value, string format)

```

Comment:
Let's make this ContainsComparableString() and try to write it to be more similar to AreComparableStrings(). Also, please use

```
/// <summary>
/// text
/// </summary>
```

For header comments.

Reply:
Really no point to the new function at all, when I look at it now.